manifests/overlays/modelserving/kustomization.yaml

```diff
@@ -17,4 +17,4 @@ images:
     digest: sha256:e4d24fd401fd4eb89b49b4ab07e0c08389384d4a672b240e98a03ad7f9ef1c85
   - name: quay.io/opendatahub/openvino_model_server
     newName: quay.io/modh/openvino_model_server
-    digest: sha256:007304a96acd654ca5133c50990c6785464fcea44304c8a846d3279b9c83a9d4
```

Comment:
@lucferbux given that the OVMS image is the same for KServe and MM, hopefully that wouldn't cause any issues here with replacing them?

Reply:
Yeah, same image is the standard case here, it's just the templates that differ